_gsocprojects/2020/project_VecGeom.md

```diff
@@ -0,0 +1,11 @@
+---
+project: VecGeom
+layout: default
+logo: vecgeom_logo.png
```

Comment:
dashes instead of underscores: vecgeom-logo.png

Reply:
That was the filename, but there is also VECGEOM-logo.png now that you say this. I will use that.